lib/flip_card.dart

```diff
@@ -61,6 +61,9 @@ class FlipCard extends StatefulWidget {
   final Fill fill;
   final CardSide side;
 
+  /// If the value is set, the flip effect will work automatically after the specified duration.
+  final Duration? autoFlipDuration;
```

Comment:
It looks like formatting is broken? Reformat or ignore this

Reply:
What is the problem here? I do not fully understand.

Reply:
Ah nevermind. I was looking at this on mobile and it looked like the formatting was weird, but it just looked like that for me.

This is completely fine!